spec/win32/unit/certstore_spec.rb

```diff
@@ -325,8 +261,7 @@
       end
       it "returns nil" do
         store = certstore.open(store_name)
-        cert_obj = store.get(thumbprint)
-        expect(cert_obj).to eql(nil)
+        expect { store.get(thumbprint, store_location: CERT_SYSTEM_STORE_CURRENT_USER, store_name: store_name) }.to raise_error(ArgumentError, "Unable to retrieve the certificate")
```

Comment:
For this and below, we should change the description to accurately talk about what is being tested - something like "raises an ArgumentError".

Reply:
Oh phooey... good catch